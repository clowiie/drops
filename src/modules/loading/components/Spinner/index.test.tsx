import React from 'react'

import { render } from '@testing-library/react'

import * as SpinnerType from '.'

describe('Spinner', () => {
  const { default: Spinner } = require('.') as typeof SpinnerType

  describe('snapshots', () => {
    it('should match snapshots', () => {
      const { container } = render(<Spinner />)

      expect(container.firstChild).toMatchSnapshot()
    })
  })
})
