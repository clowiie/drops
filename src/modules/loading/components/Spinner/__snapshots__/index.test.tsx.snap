// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner snapshots should match snapshots 1`] = `
<div
  class="sc-bdnxRM ijGDfq rounded-full w-14 h-14 animate-spin"
/>
`;
